src/apps/loan-list/list/loan-list.test.ts

```diff
@@ -481,7 +482,7 @@ describe("Loan list", () => {
       .find(".list-reservation")
       .eq(0)
       .find(".list-reservation__deadline p")
-      .should("have.text", "Due date 24-10-2022");
+      .should("have.text", "Due date 24-10-2022 08:32");
```

Comment:
This change it not related to parallel reservations but I removed `it.only` form another case in this test in order to run all tests.

To make tests pass here adding a time was necessary.